src/components/sections/DemographicsSection/DemographicsSection.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+
+import { useForm } from 'components/FormProvider';
+import SelectBox from 'components/forms/SelectBox';
+import DateField from 'components/forms/DateField';
+import TextField from 'components/forms/TextField';
+
+const ethnicityOptions = [
+  { value: 'hispanicLatino', label: 'Hispanic / Latino' },
+  { value: 'nonHispanicLatino', label: 'Non-Hispanic / Latino' },
+  { value: 'notSpecified', label: 'Not Specified' }
+];
```

Comment:
I know this is following the pattern I did in the `SelectBox` -- but I was wondering -- should we simplify this to:
```javascript
const ethnicityOptions = {
  hispanicLatino: 'Hispanic / Latino',
  nonHispanicLatino: 'Non-Hispanic / Latino' ,
  notSpecified: 'Not Specified'
};
```
The upside is it is definitely more succinct.  The downside is that if we ever wanted to add more information for each item, it doesn't support that as easily.  Anyway, it's probably fine as-is, but I thought maybe I'd think out loud.

Reply:
I like it, but I won't change it in this PR because I'm pretty sure it'll blow up my next PR haha.